docs/csharp/create-a-nested-group.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Create a nested group"
+description: How to create a nested group.
+keywords: .NET, .NET Core, C#
```

Comment:
these seem like very generic keywords. you need to have a primary one and up to 4 secondary ones. think about terms that users would use to try to find this page. I know we used them in the past and we have a lot of clean-up to do, but I'd prefer that new articles follow the best practices